src/apibuilder-cli/file_tracker.rb

```diff
@@ -0,0 +1,72 @@
+module ApibuilderCli
```

Comment:
possible to indent with two spaces?

Reply:
Definitely. Not sure why this happened - my IDE (SublimeText) is visually showing this as two spaces, not 4. I'll fix this.